README.rst

```diff
@@ -166,7 +166,8 @@ Install sqlite:
 
 .. code-block:: shell
 
-    sudo apt-get install sqlite3 libsqlite3-dev
+    sudo apt install -y sqlite3 libsqlite3-dev \
+                        libspatialite-dev libsqlite3-mod-spatialite
```

Comment:
I recommend separating these two lines and add a comment above the spatialite line to communicate that these are needed for the OpenWISP Controller integration app to run tests, we need clarify why spatialite is needed here because this will raise a few eyebrows since this app does not need GIS features.